### ISSUE TITLE
make test_all_commits fails on linux

### ISSUE THREAD

**Issue by jbenet** [jbenet/go-ipfs#927]
```
# make test_all_commits
testing all commits between origin/master..HEAD
WARNING: this will 'git rebase --exec'.
continue? [y/N] test/bin/continueyn: 8: read: arg count

test/bin/continueyn: 10: test/bin/continueyn: [[: not found
make: *** [test_all_commits] Error 1
```

**Comment by chriscool** [comment]
It might be because /bin/sh is not bash on some Linux distributions.
You might want to use /bin/bash instead.

If we want it to run on busybox, it might be better to not use bashisms though.
We could use something like:

```
case "$REPLY" in
[Yy]*) exit 0 ;;
*) exit 1 ;;
esac
```

I can write PR for that if you want.

**Comment by jbenet** [comment]
Closed in #931. Thanks!
